src/schemes/boundary/system.jl

```diff
@@ -161,13 +169,24 @@ function create_cache_boundary(::BoundaryMovement, initial_condition)
     return (; velocity, acceleration)
 end
 
+create_cache_model(viscosity, n_particles, n_dims) = (;)
+
+function create_cache_model(viscosity::ViscosityAdami, n_particles, n_dims)
+    ELTYPE = eltype(viscosity.epsilon)
+
+    wall_velocity = zeros(ELTYPE, n_dims, n_particles)
+
+    return (; wall_velocity)
```

Comment:
This should be inside `create_cache_boundary`. @LasNikas I think we decided a while ago that we want to have only one `create_boundary` call in the constructor, right?

Reply:
Yes, I wanted to remark that as well